OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,43 @@
+Example of Apache Web Server with WebLogic 12.2.1.3.0 Plugin
+=====
+This sample shows how to run a load balancer for a WebLogic cluster, from inside a container in the same network as the WLS Cluster.
+
+# How to build image
+Download file **Oracle WebLogic Server Proxy Plugins 12.2.1** [fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip](http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html) (see .download file)
+
+You may build this image with:
+
+        $ docker build -t 12213-apache .
+
+or
+
+        $ sh build.sh
+
+Oracle developers can use build_internal.sh, which set up HTTP/HTTPS proxies.
+
+        $ sh build_internal.sh
+
+# How to run container
+Run this image by calling:
+
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+The values of **WEBLOGIC_CLUSTER** must be valid, existing containers running WebLogic servers.
+
+## Example
```

Comment:
First make sure you have the WebLogic Server 12.2.1.3 install image, pull the WebLogic install image from the DockerStore store/oracle/weblogic:12.2.1.3, or build your own image oracle/weblogic:12.2.1.3-developer at [https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.3].